testing/integration/k8s/testdata/journald-input.yml

```diff
@@ -19,4 +19,5 @@ inputs:
     streams:
       - id: journald-input-id
         paths:
-          - "/opt/journald/*/*"
+          - "/opt/journald/"
```

Comment:
@belimawr doesn't this mean the change in https://github.com/elastic/beats/pull/46682 was a breaking change to configuration?

Wouldn't external users have to do the same thing?

Reply:
It's actually the opposite :disappointed:. We broke it when migrating to Journald, specifying a folder directly should have always worked.

Our [docs](https://www.elastic.co/docs/reference/beats/filebeat/filebeat-input-journald#filebeat-input-journald-paths) always stated:
> A list of paths that will be crawled and fetched. Each path can be a directory path (to collect events from all journals in a directory), or a file path. If you specify a directory, Filebeat merges all journals under the directory into a single journal and reads them.
>
> If no paths are specified, Filebeat reads from the default journal.

It's on my mental list to update the release notes/docs today to be clear about this change.

Reply:
Another option is to try detecting glob patterns and passing them as paths and only simple folders as folders. That actually might be a better solution.